### PR TITLE
Adding node-throughput prow job config

### PR DESCRIPTION
**Pull request by piyushgupta1551**
- Adding prow job config for containerd-node-throughput performance test

### CODE REVIEW

**Comment by piyushgupta1551** [comment]
Test execution ran successfully as expected but the test failed due to bad performance. 
Logs - https://storage.googleapis.com/ppc64le-kubernetes/logs/periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le/1403249172563365888/build-log.txt

PR is good for review.

**Review by mkumatag — APPROVED** [review]
/lgtm
/hold for @Rajalakshmi-Girish to review

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml [hunk 1]
```diff
@@ -176,3 +176,76 @@ periodics:
...
                 --test=ginkgo -- --parallel 10 --flake-attempts 2 --test-package-bucket k8s-release-dev --test-package-dir ci --test-package-version $K8S_BUILD_VERSION --focus-regex='\[Conformance\]' --skip-regex='\[Disruptive\]|\[Serial\]'
+  - name: periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    interval: 3h
```
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:187]: Initially, we can run every 3hrs but the frequency can be reduced later.

Comment (piyushgupta1551) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:187]: Noted.

Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:187]: Also, @mkumatag should we think of including `kubetest2` installation in all-in-one image.
We see more jobs using them.

Comment (mkumatag) [reply config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:187]: then version of that bin becomes stale and already this image is really heavy, we also need to think about making it slim.!

**Comment by mkumatag** [comment]
@piyushgupta1551 I see a message `W0611 10:05:24.283535    7414 imagepreload.go:88] No images specified. Skipping image preloading` in the logs, not sure if this is anything to do with the extra time it is taking to run! 

full log: https://prow.ppc64le-cloud.org/view/gs/ppc64le-kubernetes/logs/periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le/1403284493460049920

**Comment by piyushgupta1551** [comment]
@mkumatag , Initially I too thought so but the community also does not use image-preload in their test so have skipped it - 
`W0611 05:06:23.688180   13822 imagepreload.go:88] No images specified. Skipping image preloading`

https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-node-containerd-throughput/1403214125357600768/build-log.txt

We can add it optionally if needed by - `export NODE_PRELOAD_IMAGES=k8s.gcr.io/pause:3.1` but I dont see this impacting much. We will do a test run to see if it helps.

**Pull request merged**